packages/paima-sdk/paima-mw-core/src/errors.ts

```diff
@@ -57,6 +57,7 @@ export const enum PaimaMiddlewareErrorCode {
   INTERNAL_INVALID_POSTING_MODE,
   // only to be used as a counter
   FINAL_PAIMA_GENERIC_ERROR = 1_000_000,
+  MINA_LOGIN,
```

Comment:
Nothing should come after `FINAL_PAIMA_GENERIC_ERROR`. Probably you meant to put this after `ALGORAND_LOGIN`

The reason it's setup this way is because enums are (by default) incrementing numbers. Therefore, unless special care is taken, adding any new value to an enum is a breaking change (it shifts the value of everything that comes after it by one). To avoid this, we've properly namespaced all the enums (which is why you see some of them have explicit values), and `FINAL_PAIMA_GENERIC_ERROR` comes at the end so the users know that everything after this is a user-space error (and not a Paima Engine error)